src/social-network-provider/twitter.com/ui/injectPostInspector.tsx

```diff
@@ -0,0 +1,23 @@
+import { PostInfo } from '../../../social-network/ui'
+import { injectPostInspectorDefault } from '../../../social-network/defaults/injectPostInspector'
+
+export function injectPostInspectorAtTwitter(current: PostInfo) {
+    return injectPostInspectorDefault({
+        zipPost(node) {
+            const content = node.current.parentElement?.querySelector('[lang]')
+            Array.from(content?.children ?? []).forEach(node => {
+                const tagName = node.tagName.toLowerCase()
+                if (tagName === 'span') {
+                    node.innerHTML = node.innerHTML.replace(/—§—\s+\/\*\s+/g, '').replace(/\s+\*\/\s?/, '') // prune —§— /* */
```

Comment:
Please explain what this regex pattern is doing?

(magic code)

Reply:
<del>You can use CSS to hide the content like what we do on Facebook. (max-height: 20px).  That's safe</del>(Oh you already did that, but why do you need modify the innerHTML?). Change the innerHTML might break Twitter's React instance

Reply:
![image](https://user-images.githubusercontent.com/52657989/74306835-a5335e00-4d9e-11ea-87af-ff7591706961.png)

Just for removing the `—§— /* */` part. Otherwise it will looks like above image.

Reply:
I think it's okay to leave the `—§— /*` part here.

Reply:
Maybe we should prune it before posting or replace it with other characters to make it better look when link absents?

@neruthes Any ideas?

Reply:
My suggestion is to remove `—§—.+$`.

Reply:
@neruthes modify this content might make the enviroment unstable. I strongly recommend not to do it.

Reply:
What about hiding the whole pre-decryption raw tweet and rendering a modified version from our code?

Reply:
It can be done but I doubt if it is necessary.

Reply:
@neruthes We can just hide the `<span>` node which contains `—§— /*` and `*/`. But as I said before user content maybe hide by mistake.

Reply:
@neruthes @Jack-Works What about creating a small text or emoji between `/*` and `*/` to make it better looking. We can do this by inserting a  global style node which styles the pesudo element.

Reply:
/*🔒*/ seems good